openedxscorm/scormxblock.py

```diff
@@ -174,29 +186,50 @@ def studio_submit(self, request, _suffix):
         self.update_package_meta(package_file)
 
         # First, save scorm file in the storage for mobile clients
-        if default_storage.exists(self.package_path):
+        if scorm_storage_instance.exists(self.package_path):
             logger.info('Removing previously uploaded "%s"', self.package_path)
-            default_storage.delete(self.package_path)
-        default_storage.save(self.package_path, File(package_file))
+            scorm_storage_instance.delete(self.package_path)
+        scorm_storage_instance.save(self.package_path, File(package_file))
         logger.info('Scorm "%s" file stored at "%s"', package_file, self.package_path)
 
         # Then, extract zip file
-        if default_storage.exists(self.extract_folder_base_path):
+        if scorm_storage_instance.exists(self.extract_folder_base_path):
             logger.info(
                 'Removing previously unzipped "%s"', self.extract_folder_base_path
             )
             recursive_delete(self.extract_folder_base_path)
+
+        def unzip_member(_scorm_storage_instance,uncompressed_file,extract_folder_path, filename):
+            logger.info('Started uploading file {fname}'.format(fname=filename))
+            _scorm_storage_instance.save(
+                os.path.join(extract_folder_path, filename),
+                uncompressed_file,
+            )
+            logger.info('End uploadubg file {fname}'.format(fname=filename))
```

Comment:
This can make logs so noisy. What if we use `logger.debug` for such logs? Just a thought.